src/include/optimizer/operator_to_plan_transformer.h

```diff
@@ -13,6 +13,8 @@
 #pragma once
 
 #include "optimizer/operator_visitor.h"
+#include "properties.h"
+#include "planner/aggregate_plan.h"
 
```

Comment:
The last one is unnecessary. Will remove all the unnecessary includes in our codes.